presto-product-tests/README.md

```diff
@@ -410,11 +410,22 @@ been downloaded:
 
 ## Known issues
 
+### Port 1180 already allocated
+
 If you see the error
 ```
-ERROR: for hadoop-master  Cannot start service hadoop-master: driver failed programming external connectivity on endpoint common_hadoop-master_1 Error starting userland proxy: Bind for 0.0.0.0:1080 failed: port is already allocated
+ERROR: for hadoop-master  Cannot start service hadoop-master: driver failed programming external connectivity on endpoint common_hadoop-master_1 Error starting userland proxy: Bind for 0.0.0.0:1180 failed: port is already allocated
 ```
-You most likely have some application listening on port 1080 on either docker-machine or on your local machine if you are running docker natively.
-You can override the default socks proxy port (1080) used by dockerized Hive deployment in product tests using the
-`HIVE_PROXY_PORT` environment variable, e.g. `export HIVE_PROXY_PORT=1180`. This will run all of the dockerized tests using the custom port for the socks proxy.
-When you change the default socks proxy port (1080), you have to modify the property `hive.metastore.thrift.client.socks-proxy=hadoop-master:1080` in `conf/presto/etc/catalog/hive.properties`.
+You most likely have some application listening on port `1180` on either
+docker-machine or on your local machine if you are running docker natively.
+You can override the default socks proxy port (`1180`) used by dockerized
+Hive deployment in product tests using the `HIVE_PROXY_PORT` environment
+variable, e.g. `export HIVE_PROXY_PORT=5555`. This will run all of the
+dockerized tests using the custom port for the socks proxy.
+
+When you change the default socks proxy port (`1180`), you have to modify
+the property `hive.metastore.thrift.client.socks-proxy=hadoop-master:1180`
```

Comment:
Actually I made it a bit unclear here.
Product tests running from inside docker will have to still communicate on 1080 but from OUTSIDE of docker container (e.g. if you run product tests from your IntelliJ or connect from outside docker Presto to Hive instance [not for product tests]), then the property has to change.

To make it less confusing I'm about to start a PR changing default port inside docker soon.
Then we will have mapping 1180:1180 by default (socks proxy accessible the same way inside and outside docker) and if non-standard configuration is going to be used, then we will end up having 1180:whatever mapping.

Similar change was introduced for MySQL and Postgres.